### ISSUE TITLE
error to login and the  view the smartphone

### ISSUE THREAD

**Issue by MAT97TIA** [koush/vysor.io#495]
Yesterday it work,
today 
![Immagine](https://user-images.githubusercontent.com/64828683/81049791-29d5da00-8ebf-11ea-862a-2c1c70bda829.png)

network error , why do them ?

**Comment by MAT97TIA** [comment]
![image](https://user-images.githubusercontent.com/64828683/81050740-d19fd780-8ec0-11ea-8fd4-d7ff211f9281.png)

**Comment by VA-RedWings** [comment]
I'm having the same issue too with the newest update.  Samsung S20+

**Comment by Heretushi** [comment]
Could you check if you both have "rogue" ADB.EXE task running in the task manager ? I feel it could be related.

Looks like you are running an older version of the app (based on the color scheme). Can you get the app to update ? If your network is letting the traffic go through, it should update immediately.

I had problems due to my corporate VPN. When going on the internet straight, every error went away.

**Comment by VA-RedWings** [comment]
There is an ADB.exe task running.  How would you determine if it is 'rogue'?  I went to the file location and it is the ADB drivers I installed this morning (After I uninstalled drivers, app on PC, and app on phone).  Newest versions on my phone and windows 10 machine.

**Comment by Heretushi** [comment]
Rogue might be the wrong word. I did not mean it as a "virus" just as a "remnant that should not be there". There most probably need to be running while Vysor is running. But if you close every Vysor windows and whatnot, there should not be any abd.exe running. Sometimes, on adb.exe process stays open for whatever reason and cause problems. You could logout of your user session or reboot your computer or force close the process and it would all do the same thing : kill the process.

So, in short, before running Vysor, make sure there are no "adb.exe" process left over running and causing interference.

Best way to make extra sure : reboot the computer. ;)

**Comment by VA-RedWings** [comment]
I ended every vysor task running.  Ended the ADB task.  Doubled checked and saw no other tasks related to either.  Restarted vysor, plugged in phone, and still same error.  Ended all the tasks again, rebooted, and again same issue.

**Comment by VA-RedWings** [comment]
I'm on Version 3.0.63

**Comment by Heretushi** [comment]
Sorry it didn't help.
Before Koush asks for it, I suggest attaching a bug report to the issue. ;)

**Comment by koush** [comment]
I will need a bug report.

**Comment by VA-RedWings** [comment]
https://gist.github.com/c6ffe9731235fc1cfe402e4fa1c3011c

**Comment by koush** [comment]
@VA-RedWings Can you uninstall the universal adb drivers (did you install those?), and Vysor for Chrome, if you have that installed? And restart the computer?

I think there's some conflict with older binaries running.

**Comment by VA-RedWings** [comment]
This is what I did:

1. Uninstalled the ADB drivers
2. Checked Task Manager and ADB was still running.
3. Uninstalled Vysor
4. Checked Task Manager and ADB was running.
5.  Went to file location and it was in a Vysor folder
6.  Tried to delete folder but it wouldn't let me.
7.  Ended ADB task and Deleted folder.  Vysor folder gone.
8.  Reboot
9.  Re-Installed Vysor
10. Looked in Task Manager and ADB is there.  I didn't install Universal ADB Drivers.
11.  Plugged in phone.  Same error.

**Comment by koush** [comment]
@VA-RedWings Just to be clear, the Chrome app is not installed, or was never installed?

**Comment by VA-RedWings** [comment]
It is not installed now.  It was installed probably a few months back.  Where can I double check to ensure it is uninstalled.

**Comment by koush** [comment]
You also need to generate the bug report after you cause the bug. Leave all the windows open, and send another report. There's no information attached in the report you sent.

**Comment by koush** [comment]
You can view installed chrome apps at chrome://apps

**Comment by VA-RedWings** [comment]
Checked Chrome://apps.....No Vysor there

Bug report with error window open
https://gist.github.com/baf8ae9abc8104865ac055feb138b5c8

**Comment by VA-RedWings** [comment]
It seems to be uninstalling the APK on my phone.

**Comment by koush** [comment]
@VA-RedWings do you have a way to chat via IM? I need to debug this remotely if possible. Can send me a Vysor Share link, but don't do that in here.

**Comment by koush** [comment]
@VA-RedWings email me at koush@koushikdutta.com

**Comment by Heretushi** [comment]
Would a Vysor Share link work when he can't get Vysor to work ? :/

**Comment by koush** [comment]
@Heretushi yes actually, we ended up solving it with vysor share. vysor share creates a remote debug tunnel to the device. his hangup was on the apk installation and/or mirroring.

**Comment by ariefwijaya** [comment]
I'm having the same issue too with the newest update. Meizu m1 note

**Comment by koush** [comment]
@ariefwijaya can you try this prerelease? https://github.com/koush/vysor.io/releases/tag/v3.0.67

**Comment by ariefwijaya** [comment]
> @ariefwijaya can you try this prerelease? https://github.com/koush/vysor.io/releases/tag/v3.0.67

Nah it worked, thank you @koush  !

**Comment by koush** [comment]
@ariefwijaya the new release fixed it, or the old release started working?

**Comment by ariefwijaya** [comment]
> @ariefwijaya the new prerelease fixed it, or the old release started working?

The new prerelease fixed it, I've installed prerelease v3.0.67 as your request. @koush

**Comment by iambac777** [comment]
Resolved.   thanks!
